wandb/integration/yolov8/utils.py

```diff
@@ -0,0 +1,173 @@
+"""
+Utilities for parsing and plotting Yolo results using wandb.
+"""
+import pathlib
+from copy import deepcopy
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import cv2
+import numpy as np
+import torch
+from ultralytics.yolo.utils.ops import xywh2xyxy
+
+import wandb
+from wandb import Image
+
+
+def convert_to_wb_images(
```

Comment:
it is quite hard to read this function, could you break it down to smaller helper functions?

Reply:
adding more explicit naming and logic to distinguish between values that are ground truth vs prediction will make it more readable, as i assume some logic is repeated